libs/fairness/src/lib/Controls/ModelComparisonChart.tsx

```diff
@@ -341,15 +421,41 @@ export class ModelComparisonChart extends React.Component<
             this.props.featureBinPickerProps.selectedBinIndex,
             modelIndex,
             this.props.fairnessPickerProps.selectedFairnessKey,
-            fairnessOption.fairnessMode
+            fairnessOption.fairnessMode,
+            this.props.errorPickerProps.selectedErrorKey
           );
         });
 
       const performanceArray = (await Promise.all(performancePromises)).map(
-        (metric) => metric.global
+        (metric) => {
+          return metric.global;
+        }
+      );
+
+      const performanceBounds = (await Promise.all(performancePromises)).map(
```

Comment:
I think I wrote this in the first iteration of the PR: why not assign the result of `await Promise.all(performancePromises)` to a variable? At first glance, this looks like you're waiting on several different things even though it's the same one.